setupKAppNavLib.sh

```diff
@@ -126,7 +151,8 @@ uninstall() {
 
 install() {
     echo "########## Install started on $(date)  ##########"
-    ./install.sh $dockerID $platform
+    echo "JUNI passing $imageOption"
```

Comment:
Was this intended to be a temp echo?

Reply:
not at all. Sorry forgot to remove this one debug msg.